mne/io/meas_info.py

```diff
@@ -1227,6 +1227,16 @@ def _repr_html_(self, caption=None):
             experimenter=self.get('experimenter'),
         )
 
+    def save(self, fname):
+        """Write measurement info in fif file.
+
+        Parameters
+        ----------
+        fname : str
+            The name of the file. Should end by -info.fif.
```

Comment:
```suggestion
            The name of the file. Should end by `-info.fif`.
```

Reply:
Edited your comment to have double backticks (rst vs md code mode)